assets/common-css/navbar.css

```diff
@@ -0,0 +1,33 @@
+
+.navbar-container{
+    padding: 20px;
+    /* display: flex; */
+}
+.navbar{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.lib-name{
+    text-decoration: underline var(--secondary-color);
+    /* font-size: 1.3rem; */
+    color: var(--primary-color);
+    font-size: 1.2rem;
+}
+a{
```

Comment:
Prefer to use classnames over elements, for specificity reasons
ALso have a `reset.css` file wherein you set such defaults like `textdecoration` etc